wp-content/themes/missilethreat/archive.php

```diff
@@ -43,26 +45,137 @@
 
 </header>
 
-<div class='archive'>
+<div class='archive__content'>
 
-  <section class="archive__content">
-    
   <?php
+  if ( is_post_type_archive('missile') ) { ?>
+
+    <section class="actors">
+      <div class="archive__desc">Ballistic missiles, cruise missiles, rockets, artillery, and mortars (RAM), and even maneuvering hypersonic boost glide delivery systems now form the complicated 21st century strike complex with which U.S., allied, and partner nations must contend. Organized by country, the following represents a growing collection of information on global missile systems, with illustrations and up-to-date information on their capabilities and history.</div>
+
+      <?php
+      $args = array(
+        'post_type' => 'actors',
+        'numberposts' => -1,
+        'orderby' => 'meta_value',
+        'order' => 'ASC',
+        'meta_key' => '_actors_archive_name'
+      );
+      $actors = get_posts( $args ); ?>
+
+      <h2 class="actors__header actors__header--lower">Countries</h2>
+        <div class="actors__group actors__group-countries">
+
+          <?php
+          foreach ( $actors as $post) {
+            setup_postdata( $post );
+            $actor_icon = get_field('country_icon', $post->ID);
+            $archive_name = get_the_title();
+            $replacement_archive_name = get_post_meta( $post->ID, '_actors_archive_name', true );
+
+            if ( $replacement_archive_name ) {
+              $archive_name = $replacement_archive_name;
+            }
+
+            if ( $post->post_name === 'hezbollahs-rocket-arsenal' ) {
+              continue;
+            }
+
+            ?>
+
+            <div class="actors__container">
+              <a href="<?php echo esc_url( get_permalink() ); ?>" class="actors__link">
+                <img src="<?php echo esc_url($actor_icon['url']); ?>" alt="<?php echo esc_attr($actor_icon['alt']); ?>" class="actors__icon">
+                <?php echo $archive_name; ?>
+              </a>
+            </div>
+          <?php }  ?>
+      </div><!-- .actors__countries -->
+
+
+      <h2 class="actors__header">Sub-state Actors</h2>
+        <div class="actors__group actors__group-sub-state">
+
+          <?php
+          foreach ( $actors as $post) {
+            setup_postdata( $post );
+            $actor_icon = get_field('country_icon', $post->ID);
+            $archive_name = get_the_title();
+            $replacement_archive_name = get_post_meta( $post->ID, '_actors_archive_name', true );
+
+            if ( $replacement_archive_name ) {
+              $archive_name = $replacement_archive_name;
+            }
+
+            if ( $post->post_name !== 'hezbollahs-rocket-arsenal' ) {
```

Comment:
What is this if statement supposed to do? What is different about this particular post that it needs an exception?

Reply:
I wasn't sure how to get Hezbollah into a sub-state actors section, so I used an if statement; is there a better way to do this?